exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,7 @@
 */
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+numbers.push(4);
 /* 
```

Comment:
There is missing the 2nd part of the exercise where it asks **_to change the 1st value in the array to the number_**?
How would you do it?